report/Outline/testPiler.tex

```diff
@@ -0,0 +1,31 @@
+\section{Test Generator Plugin}
+We have chosen to only focus on Flutter development since it allows us to narrow the technical challenges into a known domain for the members of the group.
+We envision a plugin for Flutter that can be installed and will run as part of the test suite in the Flutter application.
+Since the Flutter Framework has a pre-configured test framework we envision our plugin to generate code that is compatible with the build-in test environment of Flutter.
```

Comment:
```suggestion
Since the Flutter framework has a pre-configured test framework, we envision our plugin to generate code that is compatible with the built-in test environment of Flutter.
```